app/api/public.py

```diff
@@ -30,6 +48,83 @@ def get_states_daily():
     return flask.jsonify([x.to_dict() for x in latest_daily_data])
 
 
+@api.route('/public/states/daily.csv', methods=['GET'], endpoint='states_daily')
+@api.route('/public/states/current.csv', methods=['GET'], endpoint='states_current')
+def get_states_daily_csv():
+    flask.current_app.logger.info('Retrieving States Daily')
+    include_preview = request.args.get('preview', default=False, type=inputs.boolean)
+    latest_daily_data = states_daily_query(preview=include_preview).all()
+
+    # rewrite date formats to match the old public sheet
+    reformatted_data = []
+    seen_states = set()
+    eastern_time = tz.gettz('EST')
+    for data in latest_daily_data:
+        result_dict = data.to_dict()
+
+        # for the /current endpoint, don't add the row if it's not the first occurrence of the state
```

Comment:
it might be less error-prone to have the /current endpoint just take the latest-date row for every state rather than relying on the sort here?

Reply:
Done. I now go through each row and take the one with the latest date stamp instead of relying on the sort